File: prob23/prob23.py
def longest_word(stri):
    array=stri.split(' ')
    maxi=""
    for i in stri:
        if array[i].length > maxi.length :
            maxi =  array[i]

    return maxi


strng="hsjdjhjhcj jdhjc jhdjdchjchdjchjd ejhdc sdhjd dh"
print "The longest string is {0}".format(longest_word(strng))

File: prob23/prob23.js
function longest_word(str){
  array=str.split(" ");
  max = "";
  for( i=0; i < array.length; i++){
    if(array[i].length > max.length){
      max = array[i];
    }
  }
  return max;
}


var string = "I af a fkfhkfv ff kfkvjkfvjkfvj"
document.write("The longest string is " + (longest_word(string)))
